src/Reporter/JUnitReporter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Allocine\Twigcs\Reporter;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+class JUnitReporter implements ReporterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function report(OutputInterface $output, array $violations)
+    {
+        $testsuites = new \SimpleXMLElement('<testsuites name="Twigcs"/>');
+
+        foreach ($violations as $violation) {
+            $testsuite = $testsuites->addChild('testsuite');
+            $testsuite->addAttribute('name', $violation->getFilename());
```

Comment:
This will create a testsuite for each violation. There must be one testsuite per file instead. You can see what we did in the `Allocine\Twigcs\Reporter\CheckstyleReporter` to batch violations per files.

Also, `testcase` must be the child of `testsuite`.